app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(team: Player) -> int:
+    return sum([member.get_rating() for member in team])
```

Comment:
Do not pass a list comprehension, pass a generator expression to the `sum` as mentioned in the checklist